server/src/organizations/dto/create-organization.dto.ts

```diff
@@ -0,0 +1,14 @@
+import { Organization } from '../entities/organization.entity'
+import { PartialType } from '@nestjs/mapped-types';
+
+export class CreateOrganizationDto extends PartialType(Organization){
```

Comment:
at the hackathon we talked it out and we decided we shouldn't need the `extends PartialType(Organization)` bit here
this bit says "any of the following attributes are optional" but i think for creating an organization we want to force all of these properties to be there right?

https://docs.nestjs.com/openapi/mapped-types

Reply:
I was wondering about this, myself. My bad, I thought we decided the opposite in the hackathon. But I agree. Will adjust!